Proximity Marketing/ Situated Recommendation (#46)
ICPS-MicroCity/amusement-park-analysis: doc/sections/mira-03-microcity.tex

This chapter will examine in detail the technologies that are suitable for implementing the systems described above.
Particularly the techniques available for proximity marketing, crowd density estimation and people localization in indoor and outdoor environments.

\subsection{Proximity Marketing Technologies}\label{sec:proximity-marketing-technologies}
The situated Recommendation System broadcasts notifications to devices near a certain spot.
This is the same mechanism used for Proximity Marketing techniques and seems advantageous to use the same technologies to implement such a system since there is plenty of information in the literature regarding proximity marketing.
In the following section, we will examine some of the current technologies used to implement this kind of system.

\subsubsection{Proximity Marketing with Beacons}\label{sec:proximity-marketing-with-beacons}

> Bluetooth low energy is one of the forms of Bluetooth which is designed for IoT-based applications. 

Reviewer: ```suggestion
Bluetooth low energy (BLE) is a Bluetooth specification designed for IoT-based applications, where power consumption is a critical aspect. 
```

These signals can be detected by clients that are on the lookout for that particular beacon~\cite{muddinagiri2020implementation}.
Once a person with a device with Bluetooth turned on is near the beacon, the device collects the Beacon's ID, sends it to a server that retrieves the corresponding marketing information and returns it to the user with a message.

\subsubsection{Proximity Marketing using Wi-Fi}\label{sec:proximity-marketing-wifi}
Even though proximity-driven marketing systems have mostly been developed with the use of Bluetooth beacons~\cite{mndebele2017iot}, it is possible to
enable IoT-based proximity communication running as a service using network information, exploiting Wi-Fi. This offers better value than Bluetooth in
terms of efficiency and cost-effectiveness, as it can be used to implement IoT-based proximity solutions that run on existing infrastructure.
The detection of Wi-Fi networks already provides some data about location, namely information about proximity so proximity-based rules replace location
information, where Wi-Fi hot spots work as presence sensors~\cite{dmitry2013network}.

> This paper~\cite{mndebele2017iot} for instance, presents a proximity marketing technique as a service (PMaas). The system targets people in a specific location

Reviewer: ```suggestion
This paper~\cite{mndebele2017iot} for instance, presents a proximity marketing technique as a service (PMaaS). The system targets people in a specific location
```

We believe that Wi-Fi-based calculation of visitor location within the amusement park is a promising solution in terms of both effectiveness (more
accurate location and lower consumption) and costs instead of using a traditional system like GPS.

\subsection{Situated Recommendation system}
To implement the situated Recommendation system in Mirabilandia we could assume that the technology for detecting the presence of a visitor in a certain spot could be abstracted.

> In fact, could be sufficient to associate the location of the visitor with an ID and the ID with an action. Then the system  a recommendation for the user based on the action, the position and the visitor's information.

Reviewer: ```suggestion
In fact, could be sufficient to associate the location of the visitor with an ID and the ID with an action. Then the system sends a recommendation for the user based on the action, the position and the visitor's information.
```

generates

> This way, the "situated" part of the system can be implemented in a very simple way:

Reviewer: ```suggestion
This way, the``situated'' part of the system can be implemented in a very simple way:
```

> every location of interest is previously set and every time a visitor is near sends to the visitor's wearable its id.

Reviewer: ```suggestion
every location of interest is previously set and every time a visitor is near, sends to the visitor's wearable its id.
```
